geth/node/manager.go

```diff
@@ -563,3 +565,16 @@ func (m *NodeManager) RPCServer() (*rpc.Server, error) {
 
 	return m.rpcServer, nil
 }
+
+// isNodeAvailable check if we have a node running and make sure is fully started
+func (m *NodeManager) isNodeAvailable() error {
```

Comment:
I'm thinking about renaming this into `isNodeStarted` as it may be more expressive about what it actually does. What do you think?

Reply:
It's also not entirely true because we have that `<-m.nodeStarted` floating around which only guarantees that node is started... We had a problem with naming this actually :D

Reply:
So think I that it's not entirely true. Hm. Added a refactoring note.